src/graphql/resolvers/queries/me.js

```diff
@@ -24,6 +25,33 @@ export const fieldResolvers = {
 
       return { eventId, isVotingOpen };
     },
+    favorites: async (
+      _,
+      { eventId },
+      { dataSources: { firestore, sessionLoader }, user },
+    ) => {
+      dlog('my favorite sessions called');
+
+      const favorites = await favoriteStore(firestore).findFavoritesForMember(
+        eventId,
+        user,
+      );
+      dlog('total favorites returned: %d', favorites.length);
+
+      const favoriteSessions = await Promise.all(
+        favorites.map(fav => sessionLoader.load(fav.sessionId)),
+      ).then(fs =>
+        fs
+          .filter(s => s != null)
+          .filter(s =>
+            ['ACCEPTED', 'SCHEDULED', 'CANCELLED'].includes(s.status),
```

Comment:
any way to filter this at the DB.. I guess most 99% of the sessions would fall into this status regardless, so maybe it's not a huge deal.

Reply:
@csell5 The database is retrieving the document by Id, from what I am finding you can't request the document by id AND put a where condition on it to only return if x is true. I also see no query for `docid == x && blah == y`

Reply:
ah right... 🤦 firebase.